README.md

```diff
@@ -1 +1,98 @@
-# container-resources-policy
+## Container resources policy
+
+This policy is designed to enforce constraints on the resource requirements  of
+Kubernetes containers. It follows a two-step verification process: initially
+checking whether the container has defined resource requirements, and
+subsequently ensuring that these resources fall within the permissible range
+set by the maximum resource requirements configured in the policy settings.
+
+## Configuration
+
+Users can configure the policy using the following parameters in YAML format:
+
+```yaml
+# optional
+memory:
+  defaultRequest: "100M"
+  defaultLimit: "500M"
+  maxLimit: "4G"
+# optional
+cpu:
+  defaultRequest: 100m
+  defaultLimit: 200m
+  maxLimit: 500m
+# optional
+ignoreImages: ["ghcr.io/foo/bar:1.23", "myimage", "otherimages:v1"]
+```
+
+Users can skip the optional parts of the configuration, but an empty configuration is not
+allowed. Thus, at least one of the configurations,  `cpu`, or `memory` should
+be defined including all their sub fields.  All CPU and memory configuration
+should be expressed using the [quantity
+definitions](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/)
+of Kubernetes.
+
+Any container that uses an image that matches an entry in this list will be excluded
+                from enforcement. 
+
+The `ignoreImages` configuration can be used to exclude containers from
+enforcement. Any container image that matches an entry in the list will be
+skipped. Prefix-matching can be signified with `*`. For example: `my-image-*`.
+
```

Comment:
```suggestion
It is recommended that users use the fully-qualified Docker image name (e.g. start with a domain name)
in order to avoid unexpectedly exempting images from an untrusted repository.

```